docs/index.rst

```diff
@@ -90,6 +90,8 @@ Packages
   When the problem is *not* convex, which is the case for most other uses of
   this toolbox, the initialization can play a decisive role to reach a useful
   solution.
+- :mod:`ott.neural`  .. TODO(marcocuturi): add some nice text here please.
```

Comment:
provides tools to parameterize optimal transport maps, couplings or conditional probabilities as neural networks.